### PR TITLE
To English Translations

### PR DESCRIPTION
**Pull request by Timtech4u**
Translating ES6 Quickstarts and Tips to English

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1127,283 +1248,279 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
+> In-depth learning characteristics, application production
 
 * http://ramdajs.com/
 
 * https://github.com/cujojs/most
 
 * https://lodash.com/
 
-* `fs.readdir` 问题
+* `Fs.readdir` problem
```
Comment (fundon) [inline README.md:1523]: Don't upper case.

File: README.md [hunk 2]
```diff
@@ -1127,283 +1248,279 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
 
-      let res = Promise.race([
-        delay(10, 1),
-        delay(8, 2),
-        delay(6, 3),
-        delay(4, 4),
-        delay(2, 5),
-        delay(0, 6),
+      let res = Promise.race ([
+        Delay (10, 1),
```
Comment (fundon) [inline README.md:1454]: Don't upper case.

File: README.md [hunk 3]
```diff
@@ -1127,283 +1248,279 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
+    0. * race.js *
 
-      ```js
-      function delay(value, duration = 0) {
-        return new Promise((resolve, reject) => {
-          setTimeout(() => resolve(value), duration)
+      `` `js
+      function delay (value, duration = 0) {
+        return new Promise ((resolve, reject) => {
+          SetTimeout (() => resolve (value), duration)
```
Comment (fundon) [inline README.md:1449]: Don't upper case.

File: README.md [hunk 4]
```diff
@@ -1127,283 +1248,279 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
+        Delay (8, 2),
+        Delay (6, 3),
+        Delay (4, 4),
+        Delay (2, 5),
+        Delay (0, 6),
       ])
 
-      res.then(arr => {
-        console.log(arr) // ?
+      Res.then (arr => {
```
Comment (fundon) [inline README.md:1439]: Don't upper case.

File: README.md [hunk 5]
```diff
@@ -1127,283 +1248,279 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
 
           img.url = url
         })
       }
 
-      loadImage('https://nodejs.org/static/images/logo-header.png')
-        .then(img => document.body.appendChild(img))
-        .catch(err => console.log(err))
-      ```
+      LoadImage ('https://nodejs.org/static/images/logo-header.png')
```
Comment (fundon) [inline README.md:1416]: Don't upper case.

File: README.md [hunk 6]
```diff
@@ -1127,283 +1248,279 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
+    0. * all.js *
 
-      ```js
-      function delay(value, duration = 0) {
-        return new Promise((resolve, reject) => {
-          setTimeout(() => resolve(value), duration)
+      `` `js
+      function delay (value, duration = 0) {
+        return new Promise ((resolve, reject) => {
+          SetTimeout (() => resolve (value), duration)
```
Comment (fundon) [inline README.md:1426]: Don't upper case.

File: README.md [hunk 7]
```diff
@@ -1127,283 +1248,279 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
 
           img.onload = function () {
-            resolve(img)
+            Resolve (img)
           }
 
           img.onerror = function () {
-            reject(new Error('Could not load image at ' + url))
+            Reject (new Error ('Could not load image at' + url))
           }
```
Comment (fundon) [inline README.md:1410]: Don't upper case.

File: README.md [hunk 8]
```diff
@@ -1127,283 +1248,279 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
 
-    ```js
+    `` `js
     // new
-    getProfileById(233)
-      .then(res => getFollowing(233))
-      .then(res => getFollowers(233))
-      .then(res => getStarred(233))
-      .catch(err => console.log(err))
+    GetProfileById (233)
```
Comment (fundon) [inline README.md:1387]: Don't upper case.

File: README.md [hunk 9]
```diff
@@ -1127,283 +1248,279 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
+    GetProfileById (233, (err, res) => {
       if (err) throw err
-      getFollowing(233, (err, following) => {
+      GetFollowing (233, (err, following) => {
         if (err) throw err
-        getFollowers(233, (err, followers) => {
+        GetFollowers (233, (err, followers) => {
           if (err) throw err
-          getStarred(233, (err, starred) => {
+          GetStarred (233, (err, starred) => {
```
Comment (fundon) [inline README.md:1376]: Don't upper case.

File: README.md [hunk 10]
```diff
@@ -1127,283 +1248,279 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
+    `` `js
     // old
-    getProfileById(233, (err, res) => {
+    GetProfileById (233, (err, res) => {
       if (err) throw err
-      getFollowing(233, (err, following) => {
+      GetFollowing (233, (err, following) => {
         if (err) throw err
-        getFollowers(233, (err, followers) => {
+        GetFollowers (233, (err, followers) => {
```
Comment (fundon) [inline README.md:1374]: Don't upper case.

File: README.md [hunk 11]
```diff
@@ -1127,283 +1248,279 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
-  > 面对异步编程，`callback-hell` 就是 JavaScript 给人的最大诟病！  
+  > In the face of asynchronous programming, `callback-hell` is the biggest criticism of JavaScript!
 
   * e.g.
 
-    ```js
+    `` `js
     // old
-    getProfileById(233, (err, res) => {
+    GetProfileById (233, (err, res) => {
```
Comment (fundon) [inline README.md:1370]: Don't upper case.

File: README.md [hunk 12]
```diff
@@ -1127,283 +1248,279 @@ ES6 是 **ECMAScript 6** 的简称，是 [ECMA-262 的第 6 版本](http://www.e
...
+    Array.of (1, 2, 3) // Similar to new Array (...), but without special one-arg behavior
+    [0, 0, 0] .fill (7, 1) // [0,7,7]
+    [1,2,3] .findIndex (x => x == 2) // 1
+    ["A", "b", "c"]. Entries () // iterator [0, "a"], [1, "b"], [2, "c"
+    ["A", "b", "c"]. Keys () // iterator 0, 1, 2
+    ["A", "b", "c"]. Values ​​() // iterator "a", "b", "c"
 
-    Object.assign(Point, { origin: new Point(0,0) })
-    ```
+    object.assign (Point, {origin: new Point (0,0)})
```
Comment (fundon) [inline README.md:1344]: `Object` need upper case.

**Comment by fundon** [comment]
@Timtech4u Do you want to continue the pull request?

**Comment by Timtech4u** [comment]
Hi @fundon , I am done. 
Sorry for the delay.
I hope my PR gets approved now :-D

**Pull request closed without merging**